FAO Publications.js

```diff
@@ -0,0 +1,489 @@
+{
+	"translatorID": "4883f662-29df-44ad-959e-27c9d036d165",
+	"label": "FAO Publications",
+	"creator": "Bin Liu <lieubean@gmail.com>",
+	"target": "^https?://www\\.fao\\.org/documents|publications/*",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 12,
+	"browserSupport": "gcsib",
+	"lastUpdated": "2017-09-12 09:00:08"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+	Copyright © 2017 Bin Liu
+	This file is part of Zotero.
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	//Just differentiate single and multiple. Correct itemType (either book or conferencePaper) will be determined in scrape().
+	if (url.indexOf('card') !== -1) {
+		return 'book';
+	} else {
```

Comment:
Test for multiples also that you have some results, i.e. with `getSearchResults(doc, true)`.